frontend/src/main/components/InstructorSearchTableBase.js

```diff
@@ -0,0 +1,71 @@
+import React, { Fragment } from "react";
+import { useTable, useExpanded } from "react-table";
+import { Table } from "react-bootstrap";
+
+// Stryker disable StringLiteral, ArrayDeclaration
```

Comment:
Please, no, ... this is too broad.

Use `// Stryker disable all` plus `// Stryker restore all` around the smallest section of code possible.